test/apiv2/test-apiv2

```diff
@@ -107,6 +107,22 @@ function is() {
     _show_ok 0 "$testname" "$expect" "$actual"
 }
 
+############
+#  is_not  #  Simple disequality
+############
+function is_not() {
+    local actual=$1
+    local expect_not=$2
+    local testname=$3
+
+    if [ "$actual" != "$expect_not" ]; then
+        # On success, include expected value; this helps readers understand
+        _show_ok 1 "$testname!=$expect"
+        return
+    fi
+    _show_ok -1 "$testname" "$expect" "$actual"
```

Comment:
Instead of the complicated `if/else` below, how about this instead?
```bash
    _show_ok 0 "$testname" "!= $expect" "$actual"
```

Reply:
thanks for the hint, that works!  Amended and pushed a new version.